### PR TITLE
Changed Import Statement

### PR DESCRIPTION
**Pull request by justingolden21**
Fixed [this](https://github.com/gh-conf/gh-conf-write/issues/2) issue.

Changed

<code>const readConf = require('gh-conf/gh-conf-write');</code>

to

<code>const { writeConf } = require('gh-conf/gh-conf-write');</code>

### CODE REVIEW

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
This change was supposed to be fone in [README.md](https://github.com/gh-conf/gh-conf-write#usage) file

You made the change in the wrong file.

**Comment by justingolden21** [comment]
Oops, sorry. I just undid the edit to the JS file and edited the README instead.

File: index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const writeConf = require('./write-config');
+const readConf = require('gh-conf/gh-conf-write');
```
Comment (arshadkazmi42) [inline index.js]: You need to change this to 
`const writeConf = require('./write-config');`

Then all the test cases will pass

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Looks good. 
One small change and let's get this merged after that

**Comment by justingolden21** [comment]
Ok, edited index.js back to <code>const writeConf = require('./write-config');</code>

**Review by arshadkazmi42 — APPROVED** [review]
Looks good. Thank you for working on this.

**Comment by justingolden21** [comment]
No problem. Thank you for putting up with my simple mistakes.

**Pull request merged**